R/fit_model.R

```diff
@@ -103,13 +116,9 @@ format_stan_opts <- function(sampler_opts, seed) {
       "Missing values: {.val {expected_stan_args[missing_elements]}}"
     ))
   }
-
-  # Stan sampler ------------------------------------------------------------
   EpiNow2::stan_opts(
     cores = sampler_opts[["cores"]],
     chains = sampler_opts[["chains"]],
-    # NOTE: seed gets used twice -- as the seed passed here to the Stan sampler
-    # and as the R PRNG seed for EpiNow2 initialization
```

Comment:
What's the logic behind dropping this comment?

Reply:
Code comments are better as documentation. It was moved to documentation (of the `seed` argument)